src/server/auth/server/auth.go

```diff
@@ -117,14 +123,23 @@ func NewAuthServer(etcdAddress string, etcdPrefix string) (authclient.APIServer,
 
 func (a *apiServer) activationCheck() {
 	backoff.RetryNotify(func() error {
-		watcher, err := a.admins.ReadOnly(context.Background()).Watch()
+		// Check if there are any admins present at startup
+		ro := a.admins.ReadOnly(context.Background())
+		numAdmins, err := ro.Count()
+		if err != nil {
+			return err
+		}
+		a.activated.Store(numAdmins > 0)
```

Comment:
Per our discussion this is not necessary since `Watch` already returns the current items.